src/playground-preview.ts

```diff
@@ -166,7 +210,7 @@ export class PlaygroundPreview extends PlaygroundConnectedElement {
         </mwc-icon-button>
       </div>
 
-      <div id="content">
+      <div id="content" class=${classMap({error: !!this._error})}>
```

Comment:
`classMap` already works on truthiness... is this to quiet the type-checker?

Reply:
Yeah, `Type 'undefined' is not assignable to type 'string | number | boolean'`